packages/jest-haste-map/src/lib/__tests__/docblock-test.js

```diff
@@ -16,6 +16,17 @@ const docblock = require('../docblock');
 
 describe('docblock', () => {
 
+  it('extracts valid docblock with line comment', () => {
+    const code =
+      '/**' + os.EOL + ' * @providesModule foo' + os.EOL +
+      '* // TODO: test' + os.EOL + '*/' + os.EOL +
```

Comment:
The `//` part was not necessary to break this. I saw a similar breakage that looked like this:
```
/**
 * @providesModule MyModule
 * text
 */
```